javascript-primitives-and-variables/main.js

```diff
@@ -0,0 +1,21 @@
+const fullName = 'Zach Giffen';
+const isCool = true;
+const totalPets = 4;
+
+console.log(fullName);
```

Comment:
According to step 9 these should all have labels:
**9. Log each of your variables to the console with labels.**

Reply:
eg. `console.log("fullName", fullName);

Reply:
<img width="407" alt="java stuff 4 real" src="https://github.com/Zach-Giffen/rc1023-code-solutions/assets/146271458/18f43312-9ad5-48b8-9e5b-9f379e86b447">